firstPart/4_winterIsComing.js

```diff
@@ -0,0 +1,19 @@
+const calcPairsOfGloves = (arr) => {
+    const collection = {};
+    arr.forEach(gloveColor => {
+        if (!collection.hasOwnProperty(gloveColor)) {
+            collection[gloveColor] = 0;
+        }
+        collection[gloveColor] += 1;
+    });
```

Comment:
use `reduce` instead. It is a great method to creating maps like this

Reply:
Ok.